spec/registrant_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "./lib/registrant"
+require "./lib/facility"
+
+RSpec.describe Registrant do
+  it "exist" do
+    @registrant_1 = Registrant.new("Bruce", 18, true )
+    @registrant_2 = Registrant.new("Penny", 15 )
+    expect(@registrant_1.name).to eq("Bruce")
+    expect(@registrant_1.age). to eq(18)
+    expect(@registrant_1.permit?).to eq(true)
+    expect(@registrant_1.license_data).to eq({:written => false, :license => false, :renewed => false})
+    expect(@registrant_2.name).to eq("Penny")
+    expect(@registrant_2.age).to eq(15)
+    expect(@registrant_2.permit?).to eq(false)
+    expect(@registrant_2.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    @registrant_2.earn_permit
+    expect(@registrant_2.permit?).to eq(true)
```

Comment:
```suggestion
    expect(@registrant_2.permit?).to eq(false)
```
I think you want this to be false? Your code says the registrant needs to be 16 or older. If this should actually be true, then the program code should be updated to `@age >= 15`